src/Controller/Admin/AdminThreadCommentController.php

```diff
@@ -22,17 +25,25 @@
 class AdminThreadCommentController extends Controller
 {
     use RedirectToTargetTrait;
+    use ResolveReportTrait;
 
     /**
      * Moderates a thread comment.
      *
      * @Route("/disable", methods={"GET"}, name="app_admin_thread_comment_disable")
      */
-    public function disableAction(Request $request, ThreadComment $comment, ObjectManager $manager, EventDispatcherInterface $dispatcher): Response
-    {
+    public function disableAction(
+        Request $request,
+        ThreadComment $comment,
+        ObjectManager $manager,
+        EventDispatcherInterface $dispatcher,
+        ReportRepository $reportRepository,
+        ReportManager $reportManager
+    ): Response {
         $comment->disable();
 
         $dispatcher->dispatch(Events::THREAD_COMMENT_DISABLE, new GenericEvent($comment));
```

Comment:
for example, here you can listen to THREAD_COMMENT_DISABLE event to resolve related reports